### ISSUE TITLE
Failed: ResponseRendererSpec: The response preparation logic should properly render a response with a CloseDelimited body without data

### ISSUE THREAD

**Issue by raboof** [akka/akka-http#1263]
https://jenkins.akka.io:8498/job/akka-http-nightly-against-akka-master/205//consoleFull
```
[info]   a response with a CloseDelimited body
[info]   - without data *** FAILED ***
[info]     (HTTP/1.1 200 OK

[info]     Server: akka-http/1.0.0

[info]     Date: Thu, 25 Aug 2011 09:10:29 GMT

[info]     Connection: close

[info]     Content-Type: application/json

[info]     
[info]     ,Some(false)) did not equal (HTTP/1.1 200 OK

[info]     Server: akka-http/1.0.0

[info]     Date: Thu, 25 Aug 2011 09:10:29 GMT

[info]     Connection: close

[info]     Content-Type: application/json

[info]     
[info]     ,Some(true)) (ResponseRendererSpec.scala:239)
```

**Comment by jrudolph** [comment]
Probably related to https://github.com/akka/akka-http/issues/864. That test case in the other issue has a race condition that changes the outcome if it takes too long which might be the case for this case as well.

**Comment by jrudolph** [comment]
Will also be fixed by #1372

**Comment by jrudolph** [comment]
Fixed by #1372.
